index.js

```diff
@@ -9,26 +9,72 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-const epsagon = require('epsagon');
-
-const status = require('@adobe/helix-pingdom-status');
+const { logger: setupLogger } = require('@adobe/openwhisk-action-builder/src/logging');
```

Comment:
This is probably the wrong logger. The logger you are referencing is using Loggly and intended for logging customer actions, not Helix-Services.

Reply:
no, this is the logger from openwhisk-action-builder. which can stream logs to papertrail (or loggly)